docs/developer/CHANGELOG_TEMPLATE.md

```diff
@@ -0,0 +1,67 @@
+## [Version]
+
+<!---
+* EACH ENTRY SHOULD BE A SUMMARY OF THE CHANGE; BIG CHANGES GENERALLY REQUIRE MORE EXPLANATION.
+* EACH ENTRY SHOULD BE ACCOMPANIED BY EITHER AN ISSUE NUMBER OR A PR (PREFER ISSUES OVER PRS).
```

Comment:
nit: make sure issues point to the main repo and are publicly visible

Reply:
Thanks, good clarification.